### DISCUSSION TITLE
Throw an error if not all patch keys are used

### DISCUSSION THREAD

**Discussion by tomerslice** [dolanmiu/docx#2168] [Ideas / Requests]
Hello,

I wanted to suggest a flag to check if all patch keys are used, and if some patch keys are missing data to replace with.

This validation will help to check we are not missing anything we want to replace or we are trying to replace something that is not exists.

**Comment by dolanmiu** [comment]
What should happen? Log a warning? Or throw an exception?

**Reply by tomerslice** [reply]
Exception
